pkg/v22/resource/clusterconfigmap/desired.go

```diff
@@ -50,7 +50,8 @@ func (r *StateGetter) GetDesiredState(ctx context.Context, obj interface{}) ([]*
 			Name:      key.ClusterConfigMapName(clusterConfig),
 			Namespace: key.ClusterID(clusterConfig),
 			Values: map[string]interface{}{
-				"baseDomain":   key.TenantBaseDomain(clusterConfig),
+				"baseDomain": key.DNSZone(clusterConfig),
```

Comment:
The domain was duplicated `fvu6p.k8s.fvu6p.k8s.ghost.westeurope.azure.gigantic.io`. We can just use `key.DNSZone`.